--- orig/src/ehbigint-log.h
+++ tidy/src/ehbigint-log.h
@@ -15,9 +15,11 @@
 #ifndef EHBIGINT_LOG_H
 #define EHBIGINT_LOG_H
 
+/*
 #ifdef __cplusplus
 extern "C" {
 #endif
+*/
 
 #include "ehbigint.h"		/* struct ehbigint */
 
@@ -126,8 +128,10 @@
 	} while(0)
 #endif /* EHBI_SKIP_STRUCT_NULL_CHECK */
 
+/*
 #ifdef __cplusplus
 }
 #endif
+*/
 
 #endif /* EHBIGINT_LOG_H */
--- orig/src/ehbigint-str.h
+++ tidy/src/ehbigint-str.h
@@ -15,17 +15,21 @@ License for more details.
 #ifndef EHBIGINT_STR_H
 #define EHBIGINT_STR_H
 
+/*
 #ifdef __cplusplus
 extern "C" {
 #endif
+*/
 
 #include "ehbigint.h"
 
 int ehbi_byte_to_hex_chars(unsigned char byte, char *high, char *low);
 int ehbi_hex_chars_to_byte(char high, char low, unsigned char *byte);
 
+/*
 #ifdef __cplusplus
 }
 #endif
+*/
 
 #endif /* EHBIGINT_STR_H */
--- orig/src/ehbigint-util.h
+++ tidy/src/ehbigint-util.h
@@ -15,9 +15,11 @@
 #ifndef EHBIGINT_UTIL_H
 #define EHBIGINT_UTIL_H
 
+/*
 #ifdef __cplusplus
 extern "C" {
 #endif
+*/
 
 #ifdef EHBI_NO_ALLOCA
 #include <stdlib.h>
@@ -40,8 +42,10 @@
 #endif /* ehbi_random_bytes */
 #endif /* EHBI_SKIP_IS_PROBABLY_PRIME */
 
+/*
 #ifdef __cplusplus
 }
 #endif
+*/
 
 #endif /* EHBIGINT_UTIL_H */
--- orig/src/ehbigint.h
+++ tidy/src/ehbigint.h
@@ -15,9 +15,11 @@ License for more details.
 #ifndef EHBIGINT_H
 #define EHBIGINT_H
 
+/*
 #ifdef __cplusplus
 extern "C" {
 #endif
+*/
 
 #include <stddef.h>		/* size_t */
 
@@ -435,8 +437,10 @@ enum {
 	EHBI_LAST
 };
 
+/*
 #ifdef __cplusplus
 }
 #endif
+*/
 
 #endif /* EHBIGINT_H */
--- orig/src/ehbigint-eba.h
+++ tidy/src/ehbigint-eba.h
@@ -15,9 +15,11 @@
 #ifndef EHBIGINT_EBA_H
 #define EHBIGINT_EBA_H
 
+/*
 #ifdef __cplusplus
 extern "C" {
 #endif
+*/
 
 #include "ehbigint-log.h"
 #include "ehbigint-util.h"
@@ -47,8 +49,10 @@
 #include "eba.h"
 #define Ehstr_memset Eba_memset
 
+/*
 #ifdef __cplusplus
 }
 #endif
+*/
 
 #endif /* EHBIGINT_EBA_H */
--- orig/tests/echeck.h
+++ tidy/tests/echeck.h
@@ -16,9 +16,11 @@
 #ifndef ECHECK_H
 #define ECHECK_H
 
+/*
 #ifdef __cplusplus
 extern "C" {
 #endif
+*/
 
 #include <stdlib.h>		/* for size_t */
 #include <stdio.h>		/* for FILE */
@@ -234,8 +236,10 @@
 #define check_status(val)\
 	echeck_status_m(stderr, ECHECK_FUNC, __FILE__, __LINE__, val, NULL)
 
+/*
 #ifdef __cplusplus
 }
 #endif
+*/
 
 #endif /* ECHECK_H */
--- orig/src/ehbigint-priv.h
+++ tidy/src/ehbigint-priv.h
@@ -15,9 +15,11 @@
 #ifndef EHBIGINT_PRIV_H
 #define EHBIGINT_PRIV_H
 
+/*
 #ifdef __cplusplus
 extern "C" {
 #endif
+*/
 
 #include "ehbigint.h"		/* struct ehbigint */
 
@@ -52,8 +54,10 @@
 void ehbi_internal_clear_null_struct(struct ehbigint *bi);
 void ehbi_internal_reset_bytes_used(struct ehbigint *bi, size_t from);
 
+/*
 #ifdef __cplusplus
 }
 #endif
+*/
 
 #endif /* EHBIGINT_PRIV_H */
--- orig/src/ehstr.h
+++ tidy/src/ehstr.h
@@ -20,9 +20,11 @@
 #ifndef EHSTR_H
 #define EHSTR_H
 
+/*
 #ifdef __cplusplus
 extern "C" {
 #endif
+*/
 
 #include <stddef.h>		/* size_t */
 
@@ -63,8 +65,10 @@
 #include <string.h>
 #endif
 
+/*
 #ifdef __cplusplus
 }
 #endif
+*/
 
 #endif /* EHSTR_H */
--- orig/src/ehbigint-malloc.h
+++ tidy/src/ehbigint-malloc.h
@@ -16,9 +16,11 @@
 #ifndef EHBIGINT_MALLOC_H
 #define EHBIGINT_MALLOC_H
 
+/*
 #ifdef __cplusplus
 extern "C" {
 #endif
+*/
 
 #include "ehbigint.h"
 
@@ -29,8 +31,10 @@
 struct ehbigint *ehbi_alloc_l(size_t bytes, long val);
 void ehbi_free(struct ehbigint *bi);
 
+/*
 #ifdef __cplusplus
 }
 #endif
+*/
 
 #endif /* EHBIGINT_MALLOC_H */
--- orig/src/eba.h
+++ tidy/src/eba.h
@@ -16,9 +16,11 @@
 #ifndef EBA_H
 #define EBA_H 1
 
+/*
 #ifdef __cplusplus
 extern "C" {
 #endif
+*/
 
 #ifndef EBA_SKIP_ENDIAN
 #define EBA_SKIP_ENDIAN 0
@@ -379,8 +381,10 @@
 
 #endif /* Eba_log_error0 */
 
+/*
 #ifdef __cplusplus
 }
 #endif
+*/
 
 #endif /* EBA_H */
